ui/react-components/stories/ListViewAction.stories.js

```diff
@@ -0,0 +1,115 @@
+import CustomPopup from '../components/CustomPopup/CustomPopup';
+import React, {useState} from 'react';
+import {checkinPopup, popup} from '../components/ListViewAction/ListViewAction.module.scss'
+
+export default {title: 'List View Actions'};
+
+function useListViewAction() {
+  const [show, setShow] = useState(true);
+
+  const handleClose = () => setShow(false)
+
+  return [show, handleClose]
+}
+
+export const checkin = () => {
```

Comment:
Checked the code. I have couple of suggestions:

* IMHO, it would be better to extract all the actions as independent component somewhere in `ui/react-components/components/ListViewAction/ListViewAction/<action>.jsx`.  Right now, they are defined inlined in the story itself and cannot be reused.

* We could use translated labels as well for messages like `Are you sure, you want to mark appointment as CheckedIn?` etc.

* The buttons for `checkin` and other actions are different. Could we use the same buttons even on the `check-in` action.